cmd/av/stack_tidy.go

```diff
@@ -49,14 +55,10 @@ operates on only av's internal metadata, and it won't delete the actual Git bran
 
 		for name, br := range branches {
 			if _, deleted := newParents[name]; deleted {
-				if err := meta.DeleteBranch(repo, name); err != nil {
-					return err
-				}
+				tx.DeleteBranch(name)
 				continue
 			}
-			if err := meta.WriteBranch(repo, *br); err != nil {
-				return err
-			}
+			tx.SetBranch(*br)
 		}
 		return nil
```

Comment:
Technically, you want to cancel `cu` here. But if we change the Abort to be called even after Commit (like I commented in the other PR), we can omit the cancel.